qualang_tools/plot/README.md

```diff
@@ -64,11 +64,11 @@ def ramsey_data(x, final_offset, T2, amp, initial_offset, f, phase):
             )
 
 
-tau_min = 4
-tau_max = 100
-dtau = 2
+tau_min = 32
+tau_max = 800
+dtau = 16
 taus = np.arange(tau_min, tau_max + 0.1, dtau)
-amplitude = ramsey_data(8 * taus, final_offset=0.1, T2=300, amp=1, initial_offset=0, f=10e-3, phase=0.01) + np.random.normal(
+amplitude = ramsey_data(taus, final_offset=0.1, T2=300, amp=1, initial_offset=0, f=10e-3, phase=0.01) + np.random.normal(

```

Comment:
Are these values consistent with the plot? 
I am asking because on the figure it is displayed T2= 37.5ns and f=80MHz...

Reply:
You're right, I forgot to change it and I did a small change in the example and changed the plot

Reply:
Nice !